CHANGELOG.md

```diff
@@ -1,13 +1,24 @@
 # Changelog
 
+## [0.7.1-alpha] - 11/04/2021
+
+**WARNING: Alpha releases may contain bugs and no guarantee is made about API stability. They are not recommended for production use!**
+
+### New features
+* add convenience extension method for detecting if a Shape has the Streaming trait [#512](https://github.com/awslabs/smithy-kotlin/pull/512)
+
+### Fixes
+* delegate when to set content-type to binding resolver; fix content-length for empty bodies [#525](https://github.com/awslabs/smithy-kotlin/pull/525)
+* Update regex to handle version segments greater than 9 [#521](https://github.com/awslabs/smithy-kotlin/pull/521)
```

Comment:
imo this doesn't warrent mention in public changelog but nbd either way

Reply:
My release script cares only for the header of a commit (in this case, `fix:`) and whether/not it's breaking (demarked with `!`) when deciding what to include in the changelog. If these are insufficient criteria, we should design an alternate rubric.